test/extended/builds/docker_pullsearch.go

```diff
@@ -0,0 +1,48 @@
+package builds
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds][pullsearch] docker build where the registry is not specified", func() {
+	defer g.GinkgoRecover()
+	var (
+		buildFixture = exutil.FixturePath("testdata", "builds", "test-build-search-registries.yaml")
+		oc           = exutil.NewCLI("docker-build-pullsearch")
+	)
+
+	g.Context("", func() {
+
+		g.BeforeEach(func() {
+			exutil.PreTestDump()
+		})
+
+		g.AfterEach(func() {
+			if g.CurrentGinkgoTestDescription().Failed {
+				exutil.DumpPodStates(oc)
+				exutil.DumpConfigMapStates(oc)
+				exutil.DumpPodLogsStartingWith("", oc)
+			}
+		})
+
+		g.Describe("Building from a Dockerfile whose FROM image ref does not specify the image registry", func() {
+			g.It("should create a docker build that has buildah search from our predefined list of image registries and succeed", func() {
+
+				g.By(fmt.Sprintf("calling oc create -f %q", buildFixture))
+				err := oc.Run("create").Args("-f", buildFixture).Execute()
+				o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
I believe the full `oc` invocation is logged if the test fails - `g.By` should use a more general statement:

```suggestion
				g.By("creating a BuildConfig whose base image does not have a fully qualified registry name")
				err := oc.Run("create").Args("-f", buildFixture).Execute()
				o.Expect(err).NotTo(o.HaveOccurred())
```

Reply:
suggested commit pulled in, fmt import removed, commits squashed ... thanks @adambkaplan